resolver/resolver.go

```diff
@@ -36,7 +36,7 @@ var (
 	// m is a map from scheme to resolver builder.
 	m = make(map[string]Builder)
 	// defaultScheme is the default scheme to use.
-	defaultScheme = "passthrough"
+	defaultScheme = "dns"
```

Comment:
Let's just leave this as `"passthrough"` and then the tests don't have to change.  AFAICT this value isn't ever used by gRPC anymore anyway, since you always force the default default to either `"dns"` or `"passthrough"` depending on the code path in `clientconn.go`.